src/lib/workspace/Subsetting/SubsetDownloadModal.tsx

```diff
@@ -420,7 +427,12 @@ export default function SubsetDownloadModal({
                       },
                     },
                   },
+                  icons: {
+                    inactiveColor: gray[300],
```

Comment:
because of the background color on the header cells, the inactive color would need to be at least gray[500] to pass accessibility for icons. So perhaps gray[500] for inactive and gray[900] for active?

Reply:
Thanks for checking on it! Unfortunately on my screen I think 500 and 900 are not distinguishable enough to determine active vs inactive easily. Can we settle with 400 and 900?